openstack/keppel-trivy/templates/deployment.yaml

```diff
@@ -0,0 +1,45 @@
+kind: Deployment
+apiVersion: apps/v1
+
+metadata:
+  name: trivy-server
+
+spec:
+  revisionHistoryLimit: 5
+  replicas: 1
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 1
+      maxSurge: 1
+  selector:
+    matchLabels:
+      name: trivy-server
+  template:
+    metadata:
+      labels:
+        name: trivy-server
+        alert-tier: os
+        alert-service: keppel
+    spec:
+      containers:
+        - name: trivy-server
+          image: {{ $.Values.global.ghcrIoMirror }}/aquasecurity/trivy:{{ $.Values.trivy.image_version | required ".Values.clair.image_version is not set!" }}
+          imagePullPolicy: IfNotPresent
+          command:
+            - trivy
+            - server
```

Comment:
The `--token` argument should be added to avoid unauthorized usage of the trivy-server instance.